industrial_robot_simulator/package.xml

```diff
@@ -9,15 +9,12 @@
   <author>Shaun Edwards</author>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>roscpp</build_depend>
   <build_depend>std_msgs</build_depend>
   <build_depend>sensor_msgs</build_depend>
   <build_depend>control_msgs</build_depend>
   <build_depend>trajectory_msgs</build_depend>
-  <build_depend>industrial_robot_client</build_depend>
   <build_depend>industrial_msgs</build_depend>
   <build_depend>python-rospkg</build_depend>
```

Comment:
This can be deleted. Only `run_depend` needed (see [Python module dependencies](http://docs.ros.org/indigo/api/catkin/html/howto/format1/python_module_dependencies.html)).